testdata/scripts/run_module_root.txt

```diff
@@ -0,0 +1,21 @@
+# Test that dockexec works from a module root that is a domain only
+
+[!windows] env IMAGE=busybox:1.31.1-musl
+[windows] skip 'TODO: support windows-native images in --volume flags'
+
+exec go run -exec='dockexec '$IMAGE .
+stdout 'Hello'
+
+-- go.mod --
+module foo.com
```

Comment:
could we possibly make this a surely-will-never-resolve-to-a-real-domain path? such as `domain.totallyfaketld`

Reply:
Done.